app/Console/Commands/ImportOpesCommand.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace App\Console\Commands;
+
+class ImportOpesCommand extends ImportCommand
+{
+    /**
+     * The name of the console command.
+     *
+     * @var string
+     */
+    protected $name = 'import:opes';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Import data for "Opes"';
+
+    /**
+     * Process a single value. This method can be overriden in an importer
+     * to do some kind of processing before the data are inserted into the database,
+     * e.g. converting a string value to a JSON value.
+     *
+     * @return array
+     */
+    protected function processValue(string $column, string $value)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method processValue() has a Cyclomatic Complexity of 13. The configured cyclomatic complexity threshold is 10.](https://app.codacy.com/manual/scriptotek/ub-baser/pullRequest?prid=4835963)